xdsl/ir.py

```diff
@@ -204,9 +204,13 @@ def owner(self) -> Operation:
         return self.op
 
     def __repr__(self) -> str:
-        return f"OpResult(typ={repr(self.typ)}, num_uses={repr(len(self.uses))}, " + \
-               f"op_name={repr(self.op.name)}, " + \
-               f"result_index={repr(self.result_index)}, name={repr(self.name)})"
+        return "{}<%{}: {}>(num_uses={}, op_name={})".format(
```

Comment:
Why not f-strings?

Reply:
I would agree that on large strings like here, it is way more readable.

Reply:
I actually dislike fstrings for most purposes. Purposefully splitting template and values improves readability enormously.